src/openrct2/entity/EntityRegistry.cpp

```diff
@@ -148,13 +149,17 @@ static void ResetEntityLists()
 static void ResetFreeIds()
 {
     _freeIdList.clear();
-
     _freeIdList.resize(MAX_ENTITIES);
+
     // List needs to be back to front to simplify removing
-    std::iota(std::rbegin(_freeIdList), std::rend(_freeIdList), 0);
+    auto nextId = 0;
+    std::for_each(std::rbegin(_freeIdList), std::rend(_freeIdList), [&](auto& elem) {
```

Comment:
awe man i felt so clever getting an iota in the codebase :'(

Reply:
Yeah sorry, I think @Broxzier PR adds the operator++/-- I believe it would work again then.